### PR TITLE
Featue/rm 8601 update axe version to 4 x current major

### PR DESCRIPTION
**Pull request by Frazerus**
https://re-motion.atlassian.net/browse/RM-8601

The rules deprecated in aXe core are not checked anymore, meaning that we can kick them out. The kicking out happens in an extra commit, so if we don't want that afterall, that is not much overhead :)

### CODE REVIEW

File: Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs
Comment (a-ctor) [inline Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs:138]: Using a hash set here does not make any sense if we iterate over it anyway

File: Remotion/Web/Development.WebTesting/Accessibility/AccessibilityRequirementID.cs
Comment (a-ctor) [inline Remotion/Web/Development.WebTesting/Accessibility/AccessibilityRequirementID.cs:31]: Whats the purpose of all the `//New`? Are they just hints for the review? Because we should definitely remove them before merging.

**Review by a-ctor — COMMENTED** [review]
Just a few comments regarding the implementation details. Regarding correctness I defer to mike :D

File: Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs
Comment (Frazerus) [reply Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs:138]: Those were my exact thoughts when looking at that last week 😅

File: Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs:51]: idea: add xmldoc comemnt with github url source for the respective list.

Comment (MichaelKetting) [reply Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs:51]: Add short docu to test that fails when this list changes on how to handle this.

File: Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs:53]: array

File: Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs:50]: Add TTv5

File: Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs:56]: Replace with array of regex that matches for "EN-..." and "TT\d{1-2}..."
Reasoning: a bit more precision in what we are doing.

File: Remotion/Web/Development.WebTesting/Accessibility/AccessibilityRuleID.cs
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting/Accessibility/AccessibilityRuleID.cs:497]: move comment to remarks block

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

File: Remotion/Web/Development.WebTesting.IntegrationTests/Accessibility/AccessibilityAnalyzerTest.cs
Comment (MichaelKetting) [inline Remotion/Web/Development.WebTesting.IntegrationTests/Accessibility/AccessibilityAnalyzerTest.cs:29]:
```suggestion
  /// If these tests fail after an upgrade of aXe core, new tags might have been added to the rules.
```

**Review by MichaelKetting — APPROVED** [review]

**Pull request closed without merging**
